### DISCUSSION TITLE
[solved] How do you find the stream address for raudio when you only have the web player address?

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#1510] [General]
I found three nice webradio stations but I only have the addresses to the web players and not a stream address that I can enter as Webradio in raudio. I tried poking around in Firefox/Inspect, and even capturing with Wireshark, but was not able to find anything. Can I extract that data somehow from the web player stream? 
```
http://st1.urbanrevolution.es:8000/zonaelectronica
http://chillout.zone/chillout_plus
http://chillout.zone/chillin_plus
```

**Comment by R101G** [comment]
Somewhere in Video Download Helper there is an option to extract the stream. It doesn't always work, but usually does.

**Comment by kreativkod** [comment]
@R101G Thanks for your reply! 
Actually, the links do work as they are, but I have locked down my rpi for outbound communication so the requests were blocked by my firewall. Stupid me! With Wireshark I could find the addresses queried and blocked and I added them to my webradio alias in the firewall allow rule and now the stations work.
